snc.sh

```diff
@@ -44,17 +44,17 @@ ${PODMAN} run --pull=always --rm -i quay.io/coreos/ignition-validate:release - <
 mkdir ${PWD}/${CRC_INSTALL_DIR}/tmp/
 ${PODMAN} run --pull=always --rm -v ${PWD}/${CRC_INSTALL_DIR}/tmp/:/data:Z -w /data quay.io/coreos/coreos-installer:release download -a ${ARCH} -s stable -p qemu -f qcow2.xz --decompress
 sudo mv ${CRC_INSTALL_DIR}/tmp/fedora-coreos-*-qemu.${ARCH}.qcow2 /var/lib/libvirt/images/fedora-coreos-qemu.${ARCH}.qcow2
+sudo mv -Z ${CRC_INSTALL_DIR}/fcos-config.ign /var/lib/libvirt/images/
```

Comment:
If we go this way (I'd rather not), please prefix it with `crc` or `snc`, and make sure it's cleaned up when no longer needed.

Reply:
> I'd rather not

@cfergeau is there any implication ?

 > please prefix it with crc or snc, and make sure it's cleaned up when no longer needed.

Added `snc` prefix and you want to cleanup only the `.ign` file or also the `qcow2` which is used?

Reply:
`virsh undefine` should be removing the disk images associated with the VM,, but I don't think it's going to do this for the .ign file.